agents/sorenson_dlm/dlm_agent.py

```diff
@@ -255,6 +255,49 @@ def set_current(self, session, params=None):
 
         return True, 'Set current to {}'.format(params['current'])
 
+    @ocs_agent.param('auto_acquire', default=False, type=bool)
+    @ocs_agent.param('acq_params', type=dict, default=None)
+    @ocs_agent.param('force', default=False, type=bool)
+    def init_dlm(self, session, params=None):
+        """init_dlm(auto_acquire=False, acq_params=None, force=False)
+        **Task** - Perform first time setup of the Sorenson DLM communication.
+
+        Parameters:
+            auto_acquire (bool, optional): Default is False. Starts data
+                acquisition after initialization if True.
+            acq_params (dict, optional): Params to pass to acq process if
+                auto_acquire is True.
+            force (bool, optional): Force initialization, even if already
+                initialized. Defaults to False.
+        """
+        if params is None:
+            params = {}
+
+        if self.initialized and not params.get('force', False):
```

Comment:
Using `params.get()` is no longer needed either. The default is set in the params decorator, this is just redundant. And you can safely assume `params['force']` exists and just use that directly.

So this line should read: `if self.initialized and not params['force']:`